src/main/java/org/jboss/pnc/bifrost/source/db/FinalLog.java

```diff
@@ -124,6 +121,8 @@ public static long deleteByProcessContext(long processContext, String tag, boole
             parameters.and("tag", tag);
         }
 
-        return delete(query, parameters);
+        List<FinalLog> toDelete = list(query, parameters);
```

Comment:
@michalovjan I know this is ugly, but I hit an issue with deletion with that query (https://hibernate.atlassian.net/browse/HHH-18967). This is the workaround for now. I believe there won't be too many delete statements so it's ok for now.